R/utilities-writing-report.R

```diff
@@ -232,7 +232,10 @@ renderWordReport <- function(fileName, logFolder = getwd(), createWordReport = F
     firstElement <- firstElement[1]
     # Table: caption is before table. Thus, break page is added before Table
     if (grepl(pattern = "Table", x = firstElement)) {
-      wordFileContent <- c(wordFileContent, "\\newpage")
+      # Also add a space after the table caption,
+      # If no space, some tables are not translated by pandoc
+      wordFileContent <- c(wordFileContent, "\\newpage", lineContent, "")
+      next
```

Comment:
So is this a **newline** added after the table caption? I just cannot see it watching the code...

Reply:
Yes, in this if loop, `lineContent` is the content of the line starting with "Table" as first element (e.g. "Table 1: legend for table 1"), thus `c(wordFileContent, "\\newpage", lineContent, "")` translates as:

- keep content before line starting with "Table" (`wordFileContent`)
- add a page break (`"\\newpage"`)
- add the table caption (`lineContent`)
- add a new line (`""`)

Reply:
"" is translated as a new line because we use option `sep = \n"` in the function `write` that writes the character array `wordFileContent` in a file

Reply:
I think this deserve a comment in the code. When you do funky stuff like this, explain why. We won't remember 6 months from now